src/SafeDNS/Entities/Settings.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace UKFast\SDK\SafeDNS\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $email
+ * @property array $nameservers
```

Comment:
Should we have a `Nameserver` object for these, to match other areas of the SDK?